elyra/tests/util/test_url.py

```diff
@@ -78,3 +81,36 @@ def test_invalid_file_url():
         res = unsupported_method(url)
         assert res.status_code == 405, url
         assert res.reason == "Method not allowed"
+
+
+@pytest.fixture
+def setup_env_vars():
+    # runs before test (save the value of environment variable
+    # CA_CERT_BUNDLE_PATH to avoid any contamination by tests
+    # that modify it)
+    current_ca_cert_bundle_path_value = os.environ.get("CA_CERT_BUNDLE_PATH")
+    yield
+    # runs after test (restore the value of environment variable
+    # CA_CERT_BUNDLE_PATH, if it was defined)
+    if current_ca_cert_bundle_path_value is not None:
+        os.environ["CA_CERT_BUNDLE_PATH"] = current_ca_cert_bundle_path_value
+
+
+@pytest.mark.usefixtures("setup_env_vars")
+def test_get_verify_parm():
+    """
+    Verify that method get_verify_parm works as expected:
+     - env variable CA_CERT_BUNDLE_PATH is defined
```

Comment:
TRUSTED_CA_BUNDLE_PATH

Reply:
Changed all occurrences in the code to this.